reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerOperations.java

```diff
@@ -1320,7 +1324,7 @@ static final class FailedHttpServerRequest extends HttpServerOperations {
 				ZonedDateTime timestamp,
 				ConnectionInfo connectionInfo,
 				boolean validateHeaders) {
-			super(c, listener, nettyRequest, null, connectionInfo,
+			super(c, listener, nettyRequest, null, HttpCompressionSettingsSpec.provideDefault(), connectionInfo,
```

Comment:
Specifies the default compression level supported for each compression.
- gzip, deflate: 6
- zstd: 3